components/finder.rst

```diff
@@ -192,6 +192,15 @@ You can also define your own sorting algorithm with ``sort()`` method::
         return strcmp($a->getRealPath(), $b->getRealPath());
     });
 
+You can reverse any sorting by using the ``reverseSorting()`` method::
+
+    $finder->sortByName()->reverseSorting();
+
+    $finder->sortByModifiedTime()->reverseSorting();
+
+.. versionadded:: 4.2
+    The option to reverse the sort order was introduced in Symfony 4.2.
```

Comment:
The „reverseSorting()“ method to reverse the sort order was.......

Reply:
Updated.